### PR TITLE
Add nvm support

### PR DESCRIPTION
**Pull request by matiasdominguez**
Fixes issue #239

Changes:
* Added .nvmrc file that points to the latest LTS release
* Added a line on the README with a note that the project overrides the user's Node version.

Screenshots for the change:
6f1a1faa-3b6a-4615-bac2-945e2d0a3fba

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Happy hacking!
 
 * Point browser to [localhost:5000](http://localhost:5000)
 
+* Note: the project overrides your Node version using .nvmrc which points to the latest Node LTS version.
```
Comment (jenkoian) [inline README.md]: It only overrides if they are using nvm themselves. I'd be tempted to simply have a requirements section, something like:

```
Requirements

* Node v6.11.4 (LTS) 
*Recommended to use [NVM](https://github.com/creationix/nvm)
```

Comment (matiasdominguez) [reply README.md]: Sounds good! I just changed this on the README.

Comment (matiasdominguez) [reply README.md]: BWT, I did change this, but the commit history on the PR is not updating according to my `master` branch of the fork. If a couple of hours pass and this issue still exists, I'll make a new PR.

**Comment by jenkoian** [comment]
Thanks @matiasdominguez !

**Pull request merged**